jenkins/ci.suse.de/pipelines/openstack-ardana.Jenkinsfile

```diff
@@ -28,20 +29,25 @@ pipeline {
           if ( ardana_env.startsWith("qe") || ardana_env.startsWith("qa") ) {
               env.cloud_type = "physical"
           }
+          // Use a shared workspace folder for all jobs running on the same
+          // target 'ardana_env' cloud environment
+          env.SHARED_WORKSPACE = sh (
+            returnStdout: true,
+            script: 'echo "$(dirname $WORKSPACE)/shared/${ardana_env}"'
+          ).trim()
         }
-        sh('git clone $git_automation_repo --branch $git_automation_branch automation-git')
         sh('''
+          rm -rf "$SHARED_WORKSPACE"
+          mkdir -p "$SHARED_WORKSPACE"
+          cd $SHARED_WORKSPACE
+          git clone $git_automation_repo --branch $git_automation_branch automation-git
+
           if [ -n "$github_pr" ] ; then
-            cd automation-git
-            exec scripts/jenkins/ardana/pr-update.sh
+            exec automation-git/scripts/jenkins/ardana/pr-update.sh
           fi
-        ''')
```

Comment:
Either remove the above exec or do not remove ending the sh-block here, otherwise the following may not execute.

Reply:
Good catch @JanZerebecki , thanks !